src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  const [isDisabled, setIsDisabled] = useState(true);
+  const buttonText = ["Job Opportunities", "Trainings / Workshops"];
+
+  const selectOpportunityButton = () => {
+    if (isDisabled === true) {
+      setSearchCards([...t(`home.trainings`, { returnObjects: true })]);
+      setCardsArr([...t(`home.trainings`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    } else {
+      setSearchCards([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setCardsArr([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    }
+  };
+
+  const opportunityListBtn = buttonText.map((text, index) => {
```

Comment:
this is called a btn but there are multiple btns. opportunitylistbtns?

Reply:
done